common/types/activation.go

```diff
@@ -23,8 +24,12 @@ import (
 // EpochID is the running epoch number. It's zero-based, so the genesis epoch has EpochID == 0.
 type EpochID uint32
 
-// ToBytes returns a byte-slice representation of the EpochID, using little endian encoding.
-func (l EpochID) ToBytes() []byte { return util.Uint32ToBytes(uint32(l)) }
+// Bytes returns a byte-slice representation of the EpochID, using little endian encoding.
+func (l EpochID) Bytes() []byte {
+	a := make([]byte, 4)
+	binary.LittleEndian.PutUint32(a, uint32(l))
```

Comment:
i am curious why you don't just use scale to encode this and other integer encoding.

Reply:
I didn't want to do any functional changes, I just inlined `util.Uint32ToBytes` here.

I now refactored the encoding of `EpochID` and `LayerID`. Instead of using scale sometimes and sometimes `binary.LittleEndian` it should now consistently always use scale.